pkg/gadgets/trace/tcpretrans/types/types.go

```diff
@@ -37,6 +37,8 @@ type Event struct {
 
 	SrcEndpoint eventtypes.L4Endpoint `json:"src,omitempty" column:"src"`
 	DstEndpoint eventtypes.L4Endpoint `json:"dst,omitempty" column:"dst"`
+
+	Type string `json:"type,omitempty" column:"type,minWidth:9,maxWidth:12,order:9000"`
```

Comment:
```suggestion
	Type string `json:"type,omitempty" column:"type,minWidth:9,maxWidth:10,order:9000"`
```

Could it be named differently than `Type` to avoid conflict with [Event->Type](https://github.com/inspektor-gadget/inspektor-gadget/blob/a9d9268df990e49ebd5c50bd801b2a771bbbfd77/pkg/types/types.go#L298)?